clang/test/CodeGen/RISCV/attr-hw-shadow-stack.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s -fcf-protection=return | FileCheck -check-prefix=NOTIGNORELISTED %s
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s | FileCheck -check-prefix=IGNORELISTED %s
+
+int foo(int *a) { return *a; }
+
+// CHECK: define i32 @foo(ptr %a)
+
+// IGNORELISTED-NOT: attributes {{.*}}"hw-shadow-stack"{{.*}}
+// NOTIGNORELISTED: attributes {{.*}}"hw-shadow-stack"{{.*}}
```

Comment:
What does the name 'ignore listed' mean?

Reply:
Changed to a more descriptive name